packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -96,7 +96,7 @@ const CreateAdForm = () => {
             >
                 {({ errors, handleChange, isSubmitting, isValid, setFieldValue, touched, values }) => {
                     const is_sell_advert = values.type === buy_sell.SELL;
-
+                    const preventMathAttrs = v => ['e', 'E', '+', '-'].includes(v.key) && v.preventDefault();
```

Comment:
There is a function that validates this form, `validateCreateAdForm`, please add any restrictions there. 
Also, please don't allow commas.